keras/ops/nn_test.py

```diff
@@ -974,6 +974,17 @@ def test_batch_normalization(self):
             (10, 3, 4, 5),
         )
 
+    @pytest.mark.skipif(
+        backend.backend() == "jax",
```

Comment:
Rather, use `backend.backend() not in ["tensorflow", "torch"]` since it is not implemented in the NumPy backend either

Reply:
Oh I overlooked that, fixed, thanks.